src/main/webapp/app/pages/curation/collapsible/MutationCollapsible.tsx

```diff
@@ -309,69 +312,96 @@ const MutationCollapsible = ({
-                          label="Highest level of evidence"
-                          name="level"
-                          options={[TX_LEVELS.LEVEL_NO, TX_LEVELS.LEVEL_1, TX_LEVELS.LEVEL_2]}
-                        />
-                        <RealtimeDropdownInput
-                          fieldKey={`mutations/${firebaseIndex}/tumors/${tumorIndex}/TIs/${tiIndex}/treatments/${treatmentIndex}/propagation`}
-                          label="Level of Evidence in other solid tumor types"
-                          name="propagationLevel"
-                          options={[]} // Todo
-                        />
-                        <RealtimeDropdownInput
-                          fieldKey={`mutations/${firebaseIndex}/tumors/${tumorIndex}/TIs/${tiIndex}/treatments/${treatmentIndex}/propagationLiquid`}
-                          label="Level of Evidence in other liquid tumor types"
-                          name="propagationLiquidLevel"
-                          options={[]}
-                        />
-                        <RealtimeDropdownInput
-                          fieldKey={`mutations/${firebaseIndex}/tumors/${tumorIndex}/TIs/${tiIndex}/treatments/${treatmentIndex}/fdaLevel`}
-                          label="FDA Level of Evidence"
-                          name="propagationLiquidLevel"
-                          options={[]}
-                        />
-                        <RealtimeTextAreaInput
-                          fieldKey={`mutations/${firebaseIndex}/tumors/${tumorIndex}/TIs/${tiIndex}/treatments/${treatmentIndex}/description`}
-                          inputClass={styles.textarea}
-                          label="Description of Evidence"
-                          labelIcon={
-                            <GeneHistoryTooltip
-                              historyData={parsedHistoryList}
-                              location={`${CANCER_TYPE_THERAPY_INDENTIFIER}${getMutationName(mutation)}, ${cancerTypeName}, ${
-                                treatment.name
-                              }`}
+                      <>
+                        <Collapsible
+                          className={'mt-2'}
+                          key={treatment.name_uuid}
+                          title={`Therapy: ${getTxName(drugList, treatment.name)}`}
+                          borderLeftColor={NestLevelColor[NestLevelMapping[NestLevelType.THERAPY]]}
+                          info={
+                            <TreatmentLevelSummary
+                              mutationUuid={mutation.name_uuid}
+                              cancerTypesUuid={tumor.cancerTypes_uuid}
+                              treatmentUuid={treatment.name_uuid}
                             />
                           }
-                          name="evidenceDescription"
+                          action={
+                            <>
+                              <FaEdit
+                                style={{ cursor: 'pointer' }}
```

Comment:
Is there a way we can make this style applicable to all FaEdit usage?

Reply:
We now have `OncoKBEdit`